GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/PrefetchVerbWithoutSharedCacheTests.cs

```diff
@@ -277,6 +277,33 @@ public void PrefetchCleansUpStaleTempPrefetchPacks()
             otherFilePath.ShouldBeAFile(this.fileSystem).WithContents(otherFileContents);
         }
 
+        [TestCase, Order(9)]
+        public void PrefetchCleansUpMultiPackIndexLockFile()

```

Comment:
naming nit: we're testing both midx and graph lock file cleanup. Perhaps `PrefetchCleansUpOrphanedLockFiles()` or similar?